hooks/006-add-gdm.chroot

```diff
@@ -132,6 +132,14 @@ sed -i "/^ReadWritePaths=/ a \\  /var/lib/extrausers/ \\\\" \
 # Hide gnome-terminal by default
 sed -i 's/OnlyShowIn=/NoDisplay=true\nOnlyShowIn=/g' /usr/share/applications/org.gnome.Terminal.desktop
 
+# change default wallpapers
+cp /usr/share/gnome-background-properties/ubuntu-wallpapers.xml /usr/share/gnome-background-properties/ubuntu-core-desktop-wallpapers.xml
+sed -i 's/warty-final-ubuntu.png/ubuntu-core-desktop-l.png/g' /usr/share/gnome-background-properties/ubuntu-core-desktop-wallpapers.xml
```

Comment:
these two files seem to don't exist in core-base-desktop...

Reply:
They are added under the static directory as part of this PR

Reply:
Oh, it's true!